### ISSUE TITLE
Enhance focused positioned badges

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#2133]
[Focused positioned badges rendering](https://boosted.netlify.app/docs/components/badge/#positioned) is the following:

![Screenshot 2023-07-10 at 13 46 56](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/17381666/ccb459de-8d89-4e0d-acc1-0a4aa6f0ef27)

In order to have a nice focus ring, the `<a>` should be an inline-block which would imply other class modifications.

This issue should enhance the rendering of these focused-positioned badges but should also take care of global headers that use these same kind of elements, and maybe check the impact on `/docs/utilities/position/#examples`.

A draft PR suggested a modification: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1747. It was closed to avoid having a long list of ongoing PRs.

You can maybe start from there and check the global headers use case.

**Comment by MaxLardenois** [comment]
Boosted, won't fix
